### PR TITLE
Fix SHA1 / size for agwak

### PR DESCRIPTION
**Pull request by learn-more**

### CODE REVIEW

**Review by JoachimHenze — APPROVED** [review]
not double-checked but in god we trust.

**Comment by learn-more** [comment]
This might be one of those websites that regularly updates the same file...

**Comment by whindsaks** [comment]
> This might be one of those websites that regularly updates the same file...

Which might mean we need to use archive.org.

**Comment by JoachimHenze** [comment]
Possible, that it got updated, but I do doubt that. agwak is written in Adobe Flash. A dead technology, unlikely that it is still maintained. Only tech that is even less likely to get updated versions would be something written in COBOL.

The contributor added 2 rapps-things of questionable use and quality, and proved several times, that he cannot work with git properly.
Simply removing such apps from the rapps-db should be a viable alternative to wasting time with grooming them.

**Comment by marjuanm** [comment]
Hello everyone:
I'm still a newbie to Github and the web version of it has actually given me more problems with benefits, although I generate the branch correctly when sending to its repository mix whatever the system wants, why? I don't known.

Now I am working with the desktop version of it and it has worked better for me, I think I will work only with this. I'm sorry for the inconvenience, thank you @learn-more  for your support, I'm going to be more careful with this, it was not my intention to affect the development of the project.

**Comment by JoachimHenze** [comment]
> @JoachimHenze :('

I want to apologize, I guess I might have chosen a bit too harsh words in my last comment. I definitely didn't want to insult or discourage you in any way @marjuanm . I am just a fanatic when it comes to minimizing sizes and maintenance-efforts, therefore I might have overreacted here a bit.

**Comment by marjuanm** [comment]
It's okay @JoachimHenze :) , don't worry and I appreciate that you are very patient with me, as I said, I am still a newbie in using Github, there is still a lot to improve and I hope I don't make so many mistakes.

File: agwak.txt [hunk 1]
```diff
@@ -7,5 +7,5 @@ Description = A platform game similar to Mario Bros or Kirby. We must go through
 Category = 4
 URLSite = https://www.origamihero.com/ori6/games.php
 URLDownload = https://www.origamihero.com/games/kitty3.zip
-SHA1 = CBBC5885B635C72A4FC38E46BFC713F0266CE5A9
-SizeBytes = 524667
+SHA1 = f1b16d9b3b09a0be528b7fc05efe9e03ff725ec9
+SizeBytes = 11171050
```
Comment (whindsaks) [inline agwak.txt:11]:
```suggestion
SizeBytes = 11171050
Installer = Generate
```
For automatic zip installer support

**Pull request merged**